v3data/hypes/impermanent_data.py

```diff
@@ -0,0 +1,441 @@
+
+import asyncio
+from datetime import timedelta
+
+from v3data import GammaClient, LlamaClient, UniswapV3Client
+from v3data.hypes.fees import Fees
+
+from v3data.utils import timestamp_ago
+from v3data.constants import BLOCK_TIME_SECONDS
+from v3data.hypes.fees_yield import FeesYield
+
+
+
+class ImpermanentData(FeesYield):
```

Comment:
As commented above, can we rename this (maybe ImpermanentLoss?) and remove the Data in the class name? as I have reserved that naming for classes whose function is to query and collect  raw data, separating it from the calculations.

Reply:
Isn't it an impermanent result ?  ImpermanentResult  ?

Reply:
Yeah that works, or ImpermanentDivergence?
I'll leave it up to you to decide.